architecture/additional_concepts/authorization.adoc

```diff
@@ -880,6 +880,43 @@ in their SCC set.  This allows cluster administrators to run pods as any
 user by without specifying a `RunAsUser` on the pod's `SecurityContext`.  The
 administrator may still specify a `RunAsUser` if they wish.
 
+[[rbac-to-scc]]
+==== Role-Based Access to SCCs ====
+Starting with {product-title} 3.11, you can specify Security Context Constrains as a
+resource that is handled by RBAC. This allows you to
+scope the access to your SCCs to a certain project or cluster wide. Assigning users/groups
+directly to an SCC retains cluster-wide scope.
+
+To include access to SCCs for your role, you need to specify the following rule:
```

Comment:
s/you need to/

Where do I specify this rule? Do I add it to my role definition? Do I run a command to apply it?

Reply:
You specify it in your role. I'll mention it.